packages/office-ui-fabric-react/src/components/Rating/Rating.Props.ts

```diff
@@ -53,6 +54,13 @@ export interface IRatingProps extends React.HTMLProps<HTMLElement> {
    * Optional id of label describing this instance of Rating
    */
   ariaLabelId?: string;
+
+  /**
+   * Optional setting to represent aggregate rating values on rating control.
+   * If set, rating control will be readonly rating control with partial star support.
+   * Also if set, this overrides the disabled prop
+   */
+  aggregate?: boolean;
```

Comment:
That is how we implemented this control earlier. Aggregate was shown in stars and input was enabled by default and value was floored for input selection. But this was a little confusing to the consumer as to what value does input would get when fraction is passed etc since we would display fraction but input would still be placed at the floored value.
Also from usage perspective, user will never be allowed to pass fraction and pre-populated values for individual users should never be fraction. When aggregate is used, user should not be able to input any values since fraction means value came from multiple users and user should not be allowed to change it.
This keeps the control simple to use and non confusing.
In non-aggregate mode, we still take values from the user and display the stars and allow users to change it etc just that value is floored as before this change